### PR TITLE
Throw all error while recovering account with a seed phrase

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

**Comment by marcinbodnar** [comment]
@Patrick1904 so the problem with the user you mention and seed phrase recovery has to be that `addAccessKey` throws an error but it wasn't displayed, success ware displayed instead - now it should work well, the error will be displayed.

File: src/utils/wallet.js [hunk 1]
```diff
@@ -870,6 +870,8 @@ class Wallet {
             if (lastAccount) {
                 store.dispatch(redirectTo(`/profile/${lastAccount.accountId}`, { globalAlertPreventClear: true }))
                 throw lastAccount.error
+            } else {
+                throw accountIdsError.reverse()[accountIdsError.length - 1].error
```
Comment (vgrichina) [inline src/utils/wallet.js:874]: @marcinbodnar why call `.reverse()` and then take last element? Seems like this can be just `accountIdsError[0].error`?

Comment (marcinbodnar) [reply src/utils/wallet.js]: @vgrichina I made it to be consistent with throwing `LackBalanceForState` error (first account) - but it's not necessary. Removed it.

Comment (vgrichina) [reply src/utils/wallet.js]: @marcinbodnar BTW I just realized that before fix it would also use wrong order, as `.reverse()` is happening in-place

Comment (marcinbodnar) [reply src/utils/wallet.js]: @vgrichina yeah, you're right

**Pull request merged**